src/components/PlaceList/PlaceList.tsx

```diff
@@ -1,18 +1,12 @@
-import React, { useEffect, useState } from "react";
+import React from "react";
 import { ScrollView, Dimensions, ActivityIndicator, View } from "react-native";
 import PlaceCard from "./PlaceCard";
-import type { Place } from "@googlemaps/google-maps-services-js";
 import { useNearbyPlaces } from "../NearbyPlaces/useNearbyPlaces";
 
-const deepEqual = (a: Place[], b: Place[]): boolean => {
-	return JSON.stringify(a) === JSON.stringify(b);
-};
-
 const PlaceList = (): JSX.Element => {
-	//const [places, setPlaces] = useState<Place[]>([]);
 	const { nearbyPlaces } = useNearbyPlaces();
 
-	const cardList = (nearbyPlaces || []).map((value, index) => {
+	const cardList = (nearbyPlaces?.slice(0, 5) || []).map((value, index) => {
```

Comment:
why this change? I don't think we need to limit the number of cards to 5, we only do that in nearby places for readability and relevance while strolling

Reply:
It seemed like a lot of places at once to me, but I'll change it back since it really doesn't hurt much

Reply:
@carpenterd777 still waiting on this one